internal/internal_test.go

```diff
@@ -1430,18 +1457,18 @@ func TestGetParsedBody(t *testing.T) {
 	  }`
 
 	expectedNumber := json.Number("42")
-
-	expectedObject := map[string]interface{}{}
-	expectedObject["firstname"] = "foo"
-	expectedObject["lastname"] = "bar"
-
+	expectedObject := map[string]interface{}{
+		"firstname": "foo",
+		"lastname":  "bar",
+	}
 	expectedArray := []interface{}{"hello", "opa"}
 
 	tests := map[string]struct {
-		input           *ext_authz.CheckRequest
-		want            interface{}
-		isBodyTruncated bool
-		err             error
+		input               *ext_authz.CheckRequest
+		want                interface{}
+		isBodyTruncated     bool
+		err                 error
+		protoDescriptorPath string
```

Comment:
Nit: Is `protoDescriptorPath ` used in this test ?

Reply:
Done ✔️